PHPCI/View/layout.phtml

```diff
@@ -242,8 +242,8 @@
                     </li>
                 <?php endif; ?>
 
-                <li class="archive">
-                    <a href="<?php print PHPCI_URL; ?>?archived">
+                <li class="archive<?php print (array_key_exists('archived', $_GET) ? ' active' : '') ?>">
+                    <a href="<?php print PHPCI_URL . (array_key_exists('archived', $_GET) ? '' : '?archived'); ?>">
```

Comment:
This generate !toggling feature! on archive limk and this can be not understandable from user point of view.